addons/more-links/userscript.js

```diff
@@ -0,0 +1,32 @@
+import "/libraries/linkify.min.js";
+import "/libraries/linkify-element.min.js";
+
+export default async function ({ addon }) {
+  const pageType = document.location.pathname.substr(1).split("/")[0];
+  let comments;
+
+  switch (pageType) {
+    case "users":
+      linkifyElement(document.querySelectorAll("#user-details .read-only"));
```

Comment:
This will throw, you can't pass a node list to linkify.

Reply:
No, it isn't.

Reply:
I tried it - it threw in exactly that line :(

Reply:
![image](https://user-images.githubusercontent.com/17484114/92308446-ef1aa600-ef73-11ea-98b5-095ec6441a96.png)

Reply:
Nvm, it is. Fixed in the next commit.